go.sum

```diff
@@ -45,8 +45,8 @@ github.com/Azure/go-autorest/logger v0.2.1/go.mod h1:T9E3cAhj2VqvPOtCYAvby9aBXkZ
 github.com/Azure/go-autorest/tracing v0.6.0/go.mod h1:+vhtPC754Xsa23ID7GlGsrdKBpUA79WCAKPPZVC2DeU=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
-github.com/IBM/ibm-storage-odf-operator v1.4.0 h1:OXE+umdZtrdTV31+v0dEYkst5LexxrFJUkGqv+iBCB4=
-github.com/IBM/ibm-storage-odf-operator v1.4.0/go.mod h1:TmOsuYoZIu70IMMZzdxfj1OH5EPPon3u1qzuQO9Xaxo=
+github.com/IBM/ibm-storage-odf-operator v1.3.1-0.20230821004148-24b0497905fe h1:6gonpX4fEfwlOyc8O603vXtuseTLg8Er7UTZGgWsEio=
```

Comment:
how did the version go backwards?

Reply:
the first (old line) 1.4.0 is a tag 
the second (new line) is a commit number go is calculating and generating on it's own.
It didn't go backwards but that's how go decided to call this tag... we don't have impact on it. 

On official release we will always put the official tag and not a commit / branch name